app/views/components/link.scala.html

```diff
@@ -28,7 +28,7 @@
         pClass: String = "govuk-body")(implicit messages: Messages)
 
 @link = {
- <a @{linkId.fold("")(id => s"id=$id")} class="@linkClass" href="@location">@messages(linkMessage)</a>@if(linkSentence){.}
+ <a @{linkId.fold("")(id => s"id=$id")} target = "_blank" class="@linkClass" href="@location">@messages(linkMessage)</a>@if(linkSentence){.}
```

Comment:
won't this not open links using this component in a new tab ?  I would probably wrap `target = "_blank"` in a condition